docs/libcurl/opts/CURLOPT_SSL_CTX_DATA.3

```diff
@@ -115,8 +115,8 @@ int main(void)
 }
 .fi
 .SH AVAILABILITY
-Added in 7.11.0 for OpenSSL, in 7.42.0 for wolfSSL and in 7.54.0 for
-mbedTLS. Other SSL backends are not supported.
+Added in 7.11.0 for OpenSSL, in 7.42.0 for wolfSSL, in 7.54.0 for mbedTLS,
+in 7.82.0 in BearSSL. Other SSL backends are not supported.
```

Comment:
also change this section in CURLOPT_SSL_CTX_FUNCTION.3. and feature window is currently closed, next one is likely 7.83